mailbox/store/src/test/java/org/apache/james/mailbox/store/StoreMailboxManagerAnnotationTest.java

```diff
@@ -76,9 +74,9 @@ public void setUp() throws Exception {
         storeMailboxManager = new StoreMailboxManager(mailboxSessionMapperFactory, authenticator, aclResolver, groupMembershipResolver, messageParser);
     }
 
-    @Test(expected = InvalidAnnotationException.class)
+    @Test(expected = IllegalArgumentException.class)
     public void testSetAnnotationsShouldThrowExceptionWithInvalidKey() throws Exception {
-        storeMailboxManager.updateAnnotations(mailboxPath, session, Lists.newArrayList(INVALID_KEY_ANNOTATION));
+        storeMailboxManager.updateAnnotations(mailboxPath, session, Lists.newArrayList(MailboxAnnotation.newInstance("/pri*vate/comment", "My private comment")));
```

Comment:
I think that the only code that get executed is `MailboxAnnotation.newInstace`, isn't it ?